docs/framework/wcf/feature-details/using-the-discovery-client-channel.md

```diff
@@ -28,13 +28,13 @@ When writing a WCF client application you need to know the endpoint address of t
   
- The <!--zz <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement.DiscoveryEndpoint%2A>  --> `DiscoveryEndpoint%2A>` property allows you to specify the discovery endpoint to use. Normally this is a <xref:System.ServiceModel.Discovery.UdpDiscoveryEndpoint>, but it can be any valid endpoint.  
+ The <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement.DiscoveryEndpointProvider%2A> property allows you to specify the discovery endpoint to use. Normally this is a <xref:System.ServiceModel.Discovery.UdpDiscoveryEndpoint>, but it can be any valid endpoint.  
   
- When you are creating the binding to use to communicate with the service, you must be careful to use the exact same binding as the service. The only difference is the client binding has a <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement> on the top of the stack. If service is using one of the system-provided bindings, create a new <xref:System.ServiceModel.Channels.CustomBinding> and pass in the system-provided binding to the <!--zz <xref:System.ServiceModel.CustomBinding.CustomBinding%2A> `CustomBinding` --> constructor. Then you can add the <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement> by calling `Insert` on the <!--zz <xref:System.ServiceModel.Channels.Binding.Elements%2A> --> `Elements` property.  
+ When you are creating the binding to use to communicate with the service, you must be careful to use the exact same binding as the service. The only difference is the client binding has a <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement> on the top of the stack. If service is using one of the system-provided bindings, create a new <xref:System.ServiceModel.Channels.CustomBinding> and pass in the system-provided binding to the <xref:System.ServiceModel.Channels.CustomBinding.%23ctor%2A> constructor. Then you can add the <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement%2A> by calling `Insert` on the <xref:System.ServiceModel.Channels.Binding.Elements%2A> property.  
```

Comment:
\<xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement%2A> -> \<xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement>

`<xref:System.ServiceModel.Channels.Binding.Elements%2A> doesn't exist. Wondering if it should go to this API instead: https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.channels.custombinding.elements

Reply:
><xref:System.ServiceModel.Channels.Binding.Elements%2A> doesn't exist. Wondering if it should go to this API instead: https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.channels.custombinding.elements

I think so too, yes! Thank you for pointing that out!